src/test/java/io/confluent/idesidecar/restapi/kafkarest/RecordsV3BaseSuite.java

```diff
@@ -279,7 +333,7 @@ private static Set<ProduceRequestHeader> convertResponseHeaders(
         .collect(Collectors.toSet());
   }
 
-  default void assertSame(JsonNode actual, Object expected) {
+  default void assertSame(Object expected, JsonNode actual) {
```

Comment:
This changed line is the JUnit convention, so was annoying to have it the other way around.